src/app/services/user/user.service.ts

```diff
@@ -0,0 +1,60 @@
+import { Injectable } from '@angular/core';
+import { Observable, of, throwError } from 'rxjs';
+import { share } from 'rxjs/operators';
+import { ApiService } from '../api/api.service';
+import { LoginService } from '../login/login.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
+
+  _user: any;
```

Comment:
Why are you using the underscore prefix here and for the `_loggedIn` method? If you're trying to keep them private, I'd just mark them private and remove the underscore.

Reply:
Fixed